docroot/themes/custom/mass_theme/overrides/css/contained.css

```
@@ -0,0 +1,9 @@
+
+@media (min-width: 1321px) {
```

```suggestion
@media (min-width: 911px) {
```
keep it consistent with other text elements on the page